tests/test-timber-wp-vip.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+	class TestTimberWPVIP extends TimberImage_UnitTestCase {
+
+		function testDisableCache() {
+			add_filter('timber/cache/mode', function() {
```

Comment:
This will set the filter for all subsequent tests. You should also remove the filter again.

```php
$filter = function() {
    return 'none';
};

add_filter( 'timber/cache/mode', $filter );

// Test aaaaall the things.

remove_filter( 'timber/cache/mode', $filter );